src/Libraries/CoreNodeModels/Input/FileSystem.cs

```diff
@@ -88,6 +88,12 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
                 HintPath = subNode.InnerText;
             }
         }
+
+        public override NodeInputData InputData
+        {
+            //There is not yet an appropriate input type for CoGs related to File Path or Directory Path
+            get { return null; }
```

Comment:
well, is this required for hotfix?

Reply:
really? aren't they just strings?

Reply:
I am fine leaving this... I just seemed to not make sense to me to have these nodes as inputs in the CoGs context.  Easy to remove this override if you all want me to

Reply:
if this is not related to this hotfix, can your remove it?

Reply:
Yep